tests/algorithms/test_dagger.py

```diff
@@ -86,15 +103,19 @@ def get_random_acts(obs):
         save_dir=tmpdir,
     )
     collector.reset()
-    zero_acts = np.zeros((venv.num_envs,), dtype="int")
+    zero_acts = np.zeros(
+        (venv.num_envs,) + venv.action_space.shape,
+        dtype=venv.action_space.dtype,
+    )
     obs, rews, dones, infos = collector.step(zero_acts)
     assert np.all(rews != 0)
     assert not np.any(dones)
     for info in infos:
         assert isinstance(info, dict)
-    # roll out ~5 * venv.num_envs episodes
+    # roll out 5 * venv.num_envs episodes (Pendulum-v0 has 200 timestep episodes)
```

Comment:
```suggestion
    # roll out 5 * venv.num_envs episodes (Pendulum-v1 has 200 timestep episodes)
```

(make env version consistent with test declaration)